Application/Frontend/components/Messaging/Message.tsx

```diff
@@ -1,6 +1,8 @@
 "use client";
 
 import { Group, Stack, Avatar, Text } from '@mantine/core';
+import { useUser } from '@clerk/nextjs';
```

Comment:
- Great imports for necessary dependencies such as useUser from @clerk/nextjs, as well as hooks like useState and useEffect from react.